test/integration/targets/azure_rm_sqlserver/tasks/main.yml

```diff
@@ -45,10 +45,257 @@
       - output.changed == false
       - output.state == 'Ready'
 
+# azure_rm_sqlserver_facts tests
+
+- name: Create SQL Server - 2
```

Comment:
i think it's unnecessary to create 2 service instance just to assert  facts return list

Reply:
ok, I can remove